codetransformer/instructions.py

```diff
@@ -64,7 +70,7 @@ def _vartype(self):
 
 
 class InstructionMeta(ABCMeta, matchable):
-    _marker = object()  # sentinel
+    _marker = type('_marker', (object,), {})  # sentinel
```

Comment:
It never makes it into the mro, if it is the sentinel it is replaced with object. I explicitly made this a non type to avoid this confusion. You cannot be a subclass of a non-type object.